packages/ui/src/components/DataStory/clients/WorkspacesApi.ts

```diff
@@ -2,11 +2,11 @@ import { NodeDescription } from '@data-story/core';
 import { Tree } from './Tree';
 
 export interface WorkspacesApi {
-  getTree: ({ path }) => Promise<Tree>
+  getTree: ({ path }) => Promise<Tree[]>
```

Comment:
While this is good because it mirrors arborist (?), Tree now is a somewhat misplaced name, since Tree implies a root from which recursive branches spawn. Now we have situations like `tree: Tree[]` which is also confusing. I have no suggestion for improvement as of now, but take notice of slight discomfort. Solution might be renaming, using union type or something else..

Reply:
What do you think about replacing the `Tree[]` type with `FileNodes`?

Reply:
Maybe - lets keep Tree for now and return to it later